### ISSUE TITLE
Google Lighthouse showing exclamation marks when I run a few sites - works on some though.

### ISSUE THREAD

**Issue by Kelli444** [GoogleChrome/lighthouse#16481]
### FAQ

- [x] Yes, my issue is not about [variability](https://github.com/GoogleChrome/lighthouse/blob/main/docs/variability.md) or [throttling](https://github.com/GoogleChrome/lighthouse/blob/main/docs/throttling.md).
- [x] Yes, my issue is not about a specific accessibility audit (file with [axe-core](https://github.com/dequelabs/axe-core) instead).
- [x] Yes, my issue is not answered by [other FAQs](https://github.com/GoogleChrome/lighthouse#faq).

### URL

https://kellidesign.net/

### What happened?

Hey! I have a few sites this is happening to. When I run lighthouse reports I get an exclamation mark for all the results. I have a buddy that ran it and it works for them just fine. I have a work google acct too and it gives me the same exclamation output on these few sites but not on others. Here is another example (https://integritysportandspine.com/) that shows no mixed content errors but gives the same output of the exclamation marks where the accessibility, speed, seo, etc should be

### What did you expect?

For it to give me numbers instead.

### What have you tried?

checking SSL's, BSR in wordpress, deleting any mixed content errors and replacing with another pic, incog tab, clearing site cache, disabling plugins on chrome (which doesn't make much sense anyways since it works on walmart.com and not kellidesign.net). I can run it when I download the lighthouse extension. Weird. 

### How were you running Lighthouse?

Chrome DevTools

### Lighthouse Version

idk....

### Chrome Version

135.0.7049.117 (Official Build) (64-bit) (cohort: Control) 

### OS

Windows

### Relevant log output

```sh

```
